### DISCUSSION TITLE
Inline images support

### DISCUSSION THREAD

**Discussion by mors2005** [BombusMod/BombusMod#99] [Ideas]
Hi.
Any one can not think about a new update for bombusmod with Captcha feature?
Many of bombusmod users in Nimbuzz! can not join rooms and change to other versions.
Thanks and regard.

**Comment by vitalyster** [comment]
BombusMod works perfectly with CAPTCHA Forms as described in http://xmpp.org/extensions/xep-0158.html
You can test it with rooms on conference.jabber.ru
If nimbuzz have own CAPTCHA mechanisms - give me documentation

**Comment by mors2005** [comment]
Tnx for reply.
It's about 1 week that nimbuzz change their joining question to rooms from 'math question' to loading CAPTHA, with click a link and load it in a separate ate browser in PC.
This link is a sample of new nimbuzz captcha and method of its working by using PSI+ in PC. 

http://uploadtak.com/images/v1141_Untitled_3.jpg

If its possible try to solve nimbuzz users captcha in bombusmod. [conference.nimbuzz.com]
Thanks

**Comment by vitalyster** [comment]
It's just a plain message from bot, what is the problem with BombusMod? It should show link, you should be able to open link, and enter word. Where is a problem?

**Comment by mors2005** [comment]
Yes, in bombus mod when u click the link image wont be show! Any thing!
U can test it personally, but in PC tools or nimbuzz version u can load the CAPTCHA.

**Comment by vitalyster** [comment]
Then it is a problem with your browser, not BombusMod. BombusMod just launch default browser.

**Comment by mors2005** [comment]
What?
Which browser? I'm speaking about bombusmode in phone models (java, symbian and android), all of them have this problem. They can not join to any room cuz bombusmod can not load the link of nimbuzz CAPTCHA.
Recently published 'bomboTmod' that updated for Nimbuzz CAPTCHA. But its not trusty.
Please think about this :)

**Comment by vitalyster** [comment]
Select "goto url" from message menu and you must see link on most old phones. 
I mark your issue with more appropriated title, but can not guaranteed to implement this feature for old phones. 
BTW, if you find authors of that forks where this feature already implemented - tell him his code welcomed here :)
